.pre-commit-config.yaml

```diff
@@ -13,19 +13,9 @@ repos:
         name: Check poetry (poetry-check)
         always_run: true
 
-      - id: poetry-lock
-        name: Lock poetry (poetry-lock)
-        files: "pyproject.toml"
-
-      # - id: poetry-export
-      #   name: export poetry normal and dev (poetry-export)
-      #   args: ["--with=dev", "-f", "requirements.txt", "-o", "requirements.txt"]
-      #   files: "poetry.lock"
-
-      - id: poetry-export
-        name: Export poetry docs (poetry-export)
-        args: ["--only=docs", "-f", "requirements.txt", "-o", "docs/requirements.txt"]
-        files: "poetry.lock"
+      # - id: poetry-lock
```

Comment:
Due to direct git dependencies, this step takes a while. Because we still check the `pyproject.toml` this should not be a problem to just disable it.